bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ServerConfiguration.java

```diff
@@ -2646,6 +2646,9 @@ public void validate() throws ConfigurationException {
             throw new ConfigurationException("For persisiting explicitLac, journalFormatVersionToWrite should be >= 6"
                     + "and FileInfoFormatVersionToWrite should be >= 1");
         }
+        if (getMinorCompactionInterval() <= getGcWaitTime()) {
```

Comment:
I think "=" is fine, no?

Reply:
Related to https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java#L196, i think it's better to allow "=" both `ServerConfiguration` and `GarbageCollectorThread`.

Reply:
And i found majorCompactionInterval also need to add the same verification

Reply:
I see. that looks good to me.